features/dlg_crash_info/README.md

```diff
@@ -0,0 +1,253 @@
+      - Connect the DA145xx Pro Development Kit to the host computer.
+      - UART TX on P0_6 for DA14531 (Place wire between J1:17 and J2:27) for printing functionality.
+
+      ![DA14531_connection_wires](assets/hardware_setup_531.jpg)
+
+* **Software configuration**
+
+This example requires:
+
+* SDK6.0.14 or newer (Keil project is present for armv5 and armv6 compilers)
+* **SEGGER’s J-Link** tools should be downloaded and installed.
+* **A simple serial terminal** should be installed on the PC (e.g. Putty or Teraterm).
+
+## Project Modifications
+
+In your project, make sure to exlude the default fault handler files, hardfault_handler.c and nmi_handler.c:
+
+![Excluded Files](assets/exclude_handlers.PNG)
+
+## How to run
+
+### Initial Setup
+
+For the initial setup of the project that involves linking the SDK to this SW example, please follow the Readme [here](https://github.com/dialog-semiconductor/BLE_SDK6_examples).
+
+- For the DA14531 Getting started guide you can refer to this [link](https://www.dialog-semiconductor.com/da14531-getting-started).
+
+
+### Compile & Run
+
+1. Open the project via Keil µVision 5.
+
+2. Build the project and load it to target. The project can be run either from ``System-RAM`` or ``SPI Flash``. 
+    > __Note:__ 
+    In case of SPI Flash, the [Flash Programmer](http://lpccs-docs.dialog-semiconductor.com/UM-B-083/tools/SPIFlashProgrammer.html) from SmartSnippets Toolbox should be used. Refer to the [user manual](http://lpccs-docs.dialog-semiconductor.com/UM-B-083/index.html) to get familiar with the SmartSnippets Toolbox. Or a simplified tool Smartbond Flash Programmer can be used. [SmartBond Flash Programmer](https://www.renesas.com/us/en/software-tool/smartbond-flash-programmer)
+
+3. Set up a serial terminal session by selecting the proper virtual COM port and set the port configuration as shown below:
+      - Baudrate: 115200
+      - Data: 8 bits
+      - Stop: 1 bit
+      - Parity: None
+      - Flow  control: none
+
+    > __Note:__ 
+    Refer to **Section 10** in [Get Started tutorial](http://lpccs-docs.dialog-semiconductor.com/Tutorial_SDK6/serial_port.html) for more information on enabling the UART for debugging purposes.
+
+4.  On boot, you will see a printout of number of resets along with reset issues.  Press SW2 on the motherboard to induce an error.  The terminal will provide two printouts: 1) The original boot message with no errors 2) The crash information log after the error is created.
+
+    ![Terminal_Error](assets/terminal_capture.PNG)
+
+
+5.  In addition to the UART printout, advertising data, along with a custom service is enabled to retrive the data.  A seperate tool has been created for the DA14531, as a central, to retrieve this data using GTL and python. The BLE interface is described below, and the ReadMe.md can for central_crash_info can be referrred to for operation.  
```

Comment:
Suggest linking to the specific example, or at least to the tool repo (the py_ble_manager repo will will be public soon):
https://github.com/dialog-semiconductor/py_ble_manager/tree/main/examples/central/3_central_crash_info

Reply:
I added a link in the ReadMe.md in the opening paragraph describing the BLE usage.